src/kOS/Logger.cs

```diff
@@ -58,7 +58,7 @@ public virtual void Log(Exception e)
                 message += LINE_RULE;
             }
 
-            if (SafeHouse.Config.AudibleExceptions && e is KOSException)
+            if (SafeHouse.Config.AudibleExceptions)
```

Comment:
Previously it deliberately avoided beeping non-KOSExceptions but that's silly.  If the code path gets here, that means it WILL print the exception unconditionally to the terminal, so that's the point at which a beeping is always appropriate regardless of the type of exception.  If we were suppressing an exception message it would never have gotten this far.